### ISSUE TITLE
Pithos no longer depends on gir1.2-gst-plugins-base-1.0 and it should

### ISSUE THREAD

**Issue by jkugler** [pithos/pithos#192]
I recently upgraded to Pithos 1.0.2.

After installing, I noticed that apt said

gir1.2-gst-plugins-base-1.0

was no longer needed. It didn't occur to me that the events were related, and I removed said package.  Once I did that, Pithos would no longer run, and gave this trace back:

```python
jkugler@hosanna:~$ pithos 
Traceback (most recent call last):
  File "<frozen importlib._bootstrap>", line 2135, in _find_spec
AttributeError: 'DynamicImporter' object has no attribute 'find_spec'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/bin/pithos", line 8, in <module>
    load_entry_point('pithos', 'gui_scripts', 'pithos')()
  File "/usr/lib/python3/dist-packages/pkg_resources.py", line 351, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python3/dist-packages/pkg_resources.py", line 2363, in load_entry_point
    return ep.load()
  File "/usr/lib/python3/dist-packages/pkg_resources.py", line 2088, in load
    entry = __import__(self.module_name, globals(),globals(), ['__name__'])
  File "/usr/lib/python3/dist-packages/pithos/pithos.py", line 26, in <module>
    from gi.repository import Gst, GstPbutils, GObject, Gtk, Gdk, Pango, GdkPixbuf, Gio, GLib
  File "/usr/lib/python3/dist-packages/gi/importer.py", line 53, in find_module
    'introspection typelib not found' % namespace)
ImportError: cannot import name GstPbutils, introspection typelib not found
```

Reinstalling said package fixed the error.

**Comment by gregsheremeta** [comment]
fixed in pithos_1.1.0-2ppa2 and pithos_1.0.2-2ppa1

thanks for reporting!
